CourseApp/Calculator.cs

```diff
@@ -0,0 +1,55 @@
+﻿namespace CourseApp
+{
+    public class Calculator
+    {
+        public int Add(int a, int b)
+        {
+            return a + b;
+        }
+
+        public double Add(double a, double b)
+        {
+            return a + b;
+        }
+
+        public float Add(float a, float b)
+        {
+            return a + b;
+        }
```

Comment:
By avoid of creation for a new class we can use generics here

Reply:
public T Add(T left, T right) where T : float, double, int

Reply:
Yes, but the task was set long before we met generics, so here - we may pass without generics